packages/blockchain-wallet-v4-frontend/src/data/pluginAuth/sagaRegister.ts

```diff
@@ -0,0 +1,12 @@
+import { takeLatest } from 'redux-saga/effects'
+
+import sagas from './sagas'
+import { actions } from './slice'
+
+export default ({ api, coreSagas, networks }) => {
+  const pluginLoginSaga = sagas({ api, coreSagas, networks })
+
+  return function* pluginAutoAuthSaga() {
+    yield takeLatest(actions.autoLogin, pluginLoginSaga.autoLogin)
+  }
+}
```

Comment:
why don't we just use `src/data/components/plugin` module? In that case we'd have only one single plugin related module